### DISCUSSION TITLE
Add data in pvc with RWM mode

### DISCUSSION THREAD

**Discussion by FaraSys** [longhorn/longhorn#4217] [Q&A] [answered]
Hi,
I created a pvc with RWM mode via UI. How can I upload data to this pvc?
 When I mount to a node, this disk is not visible in the host.

**Comment by mantissahz** [comment, marked as answer]
@FaraSys You could a simple pod to mount the volume and access it like this [example pod](https://longhorn.io/docs/1.3.0/references/examples/#simple-pod)

**Comment by FaraSys** [comment]
@mantissahz  That's exactly what I did. But would like to know if there is another way?

**Comment by derekbit** [comment]
@FaraSys 
Do you mean you want to upload files to the volume without a workload pod?
If yes, what's your application scenario/use case?

**Comment by FaraSys** [comment]
Hello. I wanted to compare with RWO rights. Mount to the node and upload data from the node. But I didn't find any other option. I used pod with git pull command line

**Reply by derekbit** [reply]
You can attach the volume to a node. Then, create a folder like `/mnt/disk` and mount the RWO volume by `mount /dev/longhorn/{volume name} /mnt/disk` on that node.
